docs/ops.html

```diff
@@ -3823,6 +3823,7 @@ <h3>Limitations</h3>
       version</i> (also known as the <i>inter.broker.protocol.version</i>.) Please do not attempt to do this during
       a migration, or you may break the cluster.</li>
     <li>After the migration has been finalized, it is not possible to revert back to ZooKeeper mode.</li>
+    <li>During the migration, if a ZK broker is running with multiple log directories, any directory failure will cause the broker to shutdown.</li>
```

Comment:
If I'm a user, I'd like to know - 
1) the jira link about "why" 
2) the workaround about "how"
WDTY?

Reply:
Also, I think that's the current status (v3.7.0), right? Will we fix that in the future release?

Reply:
@chia7712 great points. Updated. PTAL

@showuon No, the current plan/intention is to have that as simplification during the migration. See https://issues.apache.org/jira/browse/KAFKA-16431. Please let me know if you have any concerns with this approach. cc @cmccabe @pprovenzano